src/tools/diagnostics/analyze_resource/analyze_resource.js

```diff
@@ -24,6 +25,7 @@ async function main() {
     try {
         const resource_type = process.env.RESOURCE_TYPE;
         check_supported_resource_type(resource_type);
+        console.info('connection_basic_details', connection_basic_details);
```

Comment:
change it to dbg maybe?

Reply:
I prefer leaving it with the console.log/info/error, etc.
As we checked, I didn't use `dbg.log` in this file (it only warps for the process name and time).